lib/exercises/html_preview.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="openstax-assessment" data-uuid="<%= exercise.uuid %>">
```

Comment:
What does this look like?  Can you add a screenshot?  I'm guessing it is unstyled so doesn't look great.  Can we add some basic inline CSS here.  We should be asking any contributor to a15k to also provide basic inline CSS so that rendering on the mothership can be trivial.

Reply:
it's unstyled but uses semantic tags.  A15K supplies styles for them and makes it look "non horrible"

![screen shot 2018-07-19 at 1 37 57 pm](https://user-images.githubusercontent.com/79566/42963119-f7b17c98-8b58-11e8-8daf-4a0eed83cbc3.png)

Reply:
here's a better one:
![screen shot 2018-08-09 at 3 42 43 pm](https://user-images.githubusercontent.com/79566/43925195-3b95522e-9bec-11e8-8a56-8d27c6475ebc.png)

Reply:
![screen shot 2018-08-09 at 3 56 03 pm](https://user-images.githubusercontent.com/79566/43925387-c1094258-9bec-11e8-8c43-150724edbc0d.png)